curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657e230500602983e01fff6e.md

```diff
@@ -56,13 +56,18 @@ assert.strictEqual(scoreSpans[3].innerText, ", score = 30");
 If no straight is rolled, your `checkForStraights` function should enable the final radio button, set the value to `0`, and update the displayed text to `, score = 0`.
 
 ```js
-resetRadioOptions();
-checkForStraights([1,1,1,1,1]);
-assert.isTrue(scoreInputs[3].disabled);
-assert.isTrue(scoreInputs[4].disabled);
-assert.isFalse(scoreInputs[5].disabled);
-assert.strictEqual(scoreInputs[5].value, "0");
-assert.strictEqual(scoreSpans[5].innerText, ", score = 0");
+const assertNoStraight = (diceValuesArr) => {
+  resetRadioOptions();
+  checkForStraights(diceValuesArr);
```

Comment:
```suggestion
const assertNoStraight = (_diceValuesArr) => {
  resetRadioOptions();
  checkForStraights(_diceValuesArr);
```

This is really just a nitpick to avoid shadowing the existing `diceValuesArr` in the seed code.

Reply:
Addressed